src/components/responsive-table/rows.tsx

```diff
@@ -37,6 +37,7 @@ const Rows = ({
         hiddenColumns={hiddenColumns}
         iconCreator={iconCreator}
         expandRow={expandRow}
+        containerWidth={containerWidth}
```

Comment:
what's the `containerWidth`?

Reply:
`containerWidth` its a `fullWidth` of our table i guess